src/Orchard.Web/Modules/Orchard.CustomForms/Views/EditorTemplates/Parts.CustomForm.Fields.cshtml

```diff
@@ -42,7 +47,11 @@
     @Html.TextBoxFor(m => m.CustomFormPart.SubmitButtonText, new { @class = "text large", @Value = (Model.CustomFormPart.SubmitButtonText ?? @T("Submit").Text) })
     <span class="hint">@T("The text of the submit button.")</span>
 </fieldset>
-
+<fieldset>
+    @Html.LabelFor(m => m.CustomFormPart.PublishButtonText)
+    @Html.TextBoxFor(m => m.CustomFormPart.PublishButtonText, new { @class = "text large", @Value = (Model.CustomFormPart.PublishButtonText ?? @T("Publish").Text) })
```

Comment:
Will it be double encoded? try with some html. If it is then use a tag directly instead of the `TextBoxFor` helper, and change the other ones in this view.

Reply:
I have tried with special char and all work fine